src/mlpack/methods/ann/rnn_impl.hpp

```diff
@@ -221,6 +225,11 @@ double RNN<OutputLayerType, InitializationRuleType>::Evaluate(
     targetSize = target.n_elem / rho;
     prevRho = rho;
   }
+  else if (inputSize == 0)
```

Comment:
Ok, I think that I fixed this issue by initializing `prevRho`---or, at least, it passes the tests on my system.